### DISCUSSION TITLE
What's up with Hyas version?

### DISCUSSION THREAD

**Discussion by subidit** [thuliteio/doks#983] [Q&A] [answered]
Why does _Doks_ still use _Hyas_ version `0.5.0` when they are on `2.0.3`? 

Just being curious - is there a reason for not updating to the latest Hyas theme?

**Comment by h-enk** [comment, marked as answer]
It's because I'm still working on it. What's keeping it, is migrating Doks to use Hyas' [esbuild](https://github.com/gethyas/core/blob/main/layouts/partials/footer/esbuild.html) process.  Planned for next week.

**Comment by subidit** [comment]
Thanks for the reply. And does that mean a big update is on the way? 🚀 🚀 🚀 
Yeeah!
